cumulus/parachains/runtimes/assets/asset-hub-westend/src/genesis_config_presets.rs

```diff
@@ -107,6 +145,25 @@ pub fn get_preset(id: &PresetId) -> Option<Vec<u8>> {
 			Sr25519Keyring::well_known().map(|k| k.to_account_id()).collect(),
 			WND * 1_000_000,
 			1000.into(),
+			vec![
+			// bridged DOT
+			(
+				Location::new(2, [GlobalConsensus(rococo)]),
+				GlobalConsensusConvertsFor::<UniversalLocation, AccountId>::convert_location(
+					&Location { parents: 2, interior: [GlobalConsensus(rococo)].into() },
+				)
+				.unwrap(),
+				10000000,
+			),
+			],
+			vec![
+				// bridged DOT to Bob
+				(
+					Location::new(2, [GlobalConsensus(rococo)]),
+					get_account_id_from_seed::<sp_core::sr25519::Public>("Bob"),
```

Comment:
I think we don't need `get_account_id_from_seed` / `get_from_seed`, something like this should work maybe:
```suggestion
					Sr25519Keyring::Bob.into()
```